pkg/features/client_adapter_test.go

```diff
@@ -74,6 +74,9 @@ func TestClientAdapterAdd(t *testing.T) {
 	}
 	for name, expected := range allexpected {
 		actual, ok := all[name]
+		// ignore the version comparison
+		actual.Version = nil
```

Comment:
nit: I would change the assertion to use `cmp.Diff` and pass in the ignore option for the version field.